things/postgres/channels.go

```diff
@@ -438,6 +440,14 @@ func getNameQuery(name string) (string, string) {
 	return nq, name
 }
 
+func getSortQuery(sort string) string {
+	if sort == "name" {
+		return sort
+	}
+
+	return "id"
+}
```

Comment:
Maybe we can simplify with:
```go
if sort != "" {
    return sort
}
return "id"
```
where ID will soon be replaced by `created_at`, because ID is random.

Reply:
Let's have an extra security and only allow `name` or `created_at`